scripts/open_url_in_instance.sh

```diff
@@ -6,13 +6,14 @@ _url="$1"
 _qb_version='0.10.1'
 _proto_version=1
 _ipc_socket="${XDG_RUNTIME_DIR}/qutebrowser/ipc-$(echo -n "$USER" | md5sum | cut -d' ' -f1)"
+_qute_bin="/usr/bin/qutebrowser"
 
-if [[ -e "${_ipc_socket}" ]]; then
+if [[ -e "${_ipc_socket}" ]] && [[ `pgrep -f $_qute_bin` ]]; then
```

Comment:
I think this can be `&& pgrep -f $_qute_bin`, no need for test here.

Also, pgrep is very nice but unportable, and I'm sure some people would prefer this to be `ps` pipe to `grep -q`.

Reply:
ps | grep is kind of a pain, since grep really likes to find itself in there. : /
Simply checking the exit value of socat might do it though. I'll have another fiddle with it.